From 234fbb68ed5748c5b7fd7848f7298ff16117c13d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Alex=20R=C3=B8nne=20Petersen?= <alex@alexrp.com>
Date: Fri, 31 Oct 2025 15:45:26 +0100
Subject: [PATCH 033/133] std.Target: bump dragonfly max version to 6.4.2
X-Developer-Signature: v=1; a=openpgp-sha256; l=729; i=xtex@astrafall.org;
 h=from:subject; bh=2LT3vlBcxYxdGzL6DjlrwunDm+Wa4JkcjZ7cgCQEw7A=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn7cv/ufpf2u68++5LG2b3GNwWLVEL65EI+3l/kvTjA4
 aVUw59pHRNZGMS4GCzFFFmKDBu8WXXS+UWXlcvCzGFlAhkiLdLAwMDAwMLAl5uYV2qkY6Rnqm2o
 Z2ioY6RjxMDFKQBTXSDP8D8zcvGCvV89nVkr7jYs6XRPefi/OlUuTmqOesWlgPnBGbaMDJOe2TX
 VX93h3dta1b50FWNE7tK9em23ZWc+VnvLf/NgKwsA
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

(cherry picked from commit 3801e6366f961e8d182f4012c50acd919dd7ce67)
---
 lib/std/Target.zig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/std/Target.zig b/lib/std/Target.zig
index c4819f58759a..7f2a7a7a7b40 100644
--- a/lib/std/Target.zig
+++ b/lib/std/Target.zig
@@ -502,7 +502,7 @@ pub const Os = struct {
                 .dragonfly => .{
                     .semver = .{
                         .min = .{ .major = 6, .minor = 0, .patch = 0 },
-                        .max = .{ .major = 6, .minor = 4, .patch = 1 },
+                        .max = .{ .major = 6, .minor = 4, .patch = 2 },
                     },
                 },
                 .freebsd => .{
-- 
2.52.0

